### PR TITLE
Add a PyGMT tutorial on 2021-03-16

### PR DESCRIPTION
**Pull request by chey0007**

### CODE REVIEW

File: source/groupmeeting/workshops.rst [hunk 1]
```diff
@@ -3,6 +3,15 @@ Workshops
 
 Some workshops hosted in MIG, with PPTs in MIG Google Drive (``groupmeeting.NTU/workshops.MIGG``).
 
+PyGMT Workshop, Introduction to Markdown, VS Code and Git
+---------------------------------------------------------
```
Comment (core-man) [inline source/groupmeeting/workshops.rst:7]:
```suggestion
PyGMT Workshop
--------------
```

File: source/groupmeeting/workshops.rst [hunk 2]
```diff
@@ -3,6 +3,15 @@ Workshops
...
+
+An introduction to PyGMT:
```
Comment (core-man) [inline source/groupmeeting/workshops.rst:9]:
```suggestion
`An introduction to PyGMT <https://github.com/MIGG-NTU/PyGMT2021>`__:
```

File: source/groupmeeting/workshops.rst [hunk 3]
```diff
@@ -3,6 +3,15 @@ Workshops
...
+
+- Github repository: https://github.com/MIGG-NTU/PyGMT2021 which contains the installation dependencies and the tutorial file (Jupyter Lab / Jupyter Notebook).
+- Installation instrutions to PyGMT: https://www.pygmt.org/latest/install.html
+- Tutorials regarding basic plots and notes regarding the command-line in GMT compared to PyGMT are discussed.
```
Comment (core-man) [inline source/groupmeeting/workshops.rst:13]:
```suggestion
- Basic plots
- Difference between the command-line in GMT and PyGMT
- How to use non-wrapped GMT commands
- Some reference tutorial
```

File: source/groupmeeting/workshops.rst [hunk 4]
```diff
@@ -3,6 +3,15 @@ Workshops
...
+
```
Comment (core-man) [inline source/groupmeeting/workshops.rst:14]:
```suggestion

`Introduction to some useful tools <https://github.com/MIGG-NTU/PyGMT2021/issues/2>`__

- Markdown
- VS Code
- Git
```

**Comment by core-man** [comment]
@chey0007 I made some comments. You can click `commit suggestion` to accept those comments or you may revise them at the `chey0007-patch-1` branch.

**Comment by core-man** [comment]
Please change the title of this PR a little descriptive, e.g., `Add a PyGMT tutorial on 2021-03-16`. We don't know what this PR is if we use an title like "Update workshop.rst`, right?

See how the title looks like in the main branch: https://github.com/MIGG-NTU/MIG_Docs/commits/master

@HouseJaay also did not use a descriptive title for some PRs.

**Comment by chey0007** [comment]
Noted, I have revised it based on the suggestions

**Comment by chey0007** [comment]
noted, I didnt realize I could actually name the changes initially. Will take note when I do future changes next time, thanks!

**Review by core-man — APPROVED** [review]

**Comment by chey0007** [comment]
I now see the area to edit the title now, thanks again.

**Comment by core-man** [comment]
I have changed the title. Please click the "Edit" button at right to change the PR title in the future when you want a better descriptive one.

Screenshot 2021-03-18 at 11 34 29 PM

---

See the title looks like:

Screenshot 2021-03-18 at 11 36 23 PM

**Comment by core-man** [comment]
Besides, when you click the "Merge and Squash" button, you may also clear up some unuseful information:

Screenshot 2021-03-18 at 11 37 41 PM

You may only keep one line of "Co-authored-by: Yao Jiayuan <coreman.seism@gmail.com>" and remove all other commit information, e.g., `Update worksop.rst`, `Update source/groupmeeting/workshops.rst`.

**Pull request merged**